.github/workflows/ci.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: Unit tests
     strategy:
       matrix:
-        python-version: [3.7]
+        python-version: [3.7, 3.8]
```

Comment:
Can we change this to 3.10? That's the new version that we are migrating to.

Reply:
Done!